common/core/web/keyboard-processor/src/text/kbdInterface.ts

```diff
@@ -958,5 +958,47 @@ namespace com.keyman.text {
 
       return behavior;
     }
+
+    // Publishes the KeyboardInterface's shorthand API names.
+    // Note that this may need to be called multiple times; a keyboard-processor consumer
+    // may extend or overwrite some of the callbacks after this method's initial call.
```

Comment:
I don't really understand this comment. What situation are you envisaging where a keyboard-processor consumer would fiddle with these? (monkey patching is always possible with JS after all so no need to note that unless there are specific circumstances which make it important...)

Reply:
Please refer to https://github.com/keymanapp/keyman/pull/2994#discussion_r417703005 and https://github.com/keymanapp/keyman/pull/2994#discussion_r417702108.  The latter is the specific reason that this comment is in place.

The first site is always called, but the second site only exists within full, DOM-aware KMW.

Reply:
So, kmwapi.ts is the place
> where a keyboard-processor consumer would fiddle with these[.]

That is, DOM-aware KeymanWeb is that consumer.

Reply:
I see. Can you update the comment accordingly to clarify that the DOM aware KMW will rewrite some of these? It'll help future maintainers. Thanks.

(FWIW, explanations within PRs are helpful only for the review of that PR -- so where possible, explain by adding comments to the code -- because if one reviewer has questions, other readers of the code may as well. Reminding myself here...)

Reply:
Sure.  Guess that wasn't clear enough before, so I'll make it extra-clear.  Feels a bit weird explicitly referencing DOM-aware KMW specifics from keyboard-processor in function documentation, but it is what it is.